src/main/kotlin/com/devfloor/untitled/comment/presentation/CommentController.kt

```diff
@@ -1,20 +1,19 @@
 package com.devfloor.untitled.comment.presentation
 
-import com.devfloor.untitled.comment.application.CommentFacade
-import com.devfloor.untitled.comment.application.CommentResponse
+import com.devfloor.untitled.comment.application.CommentModifyRequest
+import com.devfloor.untitled.comment.application.CommentService
 import org.springframework.http.HttpStatus
-import org.springframework.web.bind.annotation.GetMapping
-import org.springframework.web.bind.annotation.PathVariable
-import org.springframework.web.bind.annotation.ResponseStatus
-import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.bind.annotation.*
 
 @RestController
 class CommentController(
-    private val commentFacade: CommentFacade,
+    private val commentService: CommentService,
 ) {
-    @GetMapping(value = ["/articles/{articleId}/comments"])
-    @ResponseStatus(value = HttpStatus.OK)
-    fun showAllByArticleId(@PathVariable articleId: Long): List<CommentResponse> {
-        return commentFacade.showAllByArticleId(articleId)
-    }
+    @PutMapping(value = ["/articles/{articleId}/comments/commentId"])
+    @ResponseStatus(value = HttpStatus.NO_CONTENT)
+    fun modifyByCommentId(
+        @PathVariable articleId: Long,
+        @PathVariable commentId: Long,
+        @RequestBody request: CommentModifyRequest
+    ) = commentService.modifyByArticleId(articleId, commentId, request)
 }
```

Comment:
이 부분 참고하시면 좋을거같아요.! 한줄로 처리되지 않는 경우 아래와 같은 형태를 따르라고 하는거같네요.!
https://kotlinlang.org/docs/coding-conventions.html#functions
```suggestion
        @RequestBody request: CommentModifyRequest
    ) {
        return commentService.modifyByArticleId(articleId, commentId, request)
    }
}
```

Reply:
return이 없어 return 빼고 블록바디로 변경할게욥!